ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1,5 +1,108 @@
 presubmits:
   openshift/installer:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-installer-master-e2e
+    optional: true
+    rerun_command: /test e2e
+    spec:
+      containers:
+      - args:
+        - -c
+        - |
+          set -eux
+          target=$(awk < /usr/local/e2e-targets \
+              --assign "r=$RANDOM" \
+              'BEGIN { r /= 32767 } (r -= $1) <= 0 { print $2; exit }')
```

Comment:
@bbguimaraes  curious - what does 32767 represent? in other words how you picked this value ?

Reply:
https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-RANDOM

> RANDOM
>
>    Each time this parameter is referenced, a random integer between 0 and 32767 is generated. Assigning a value to this variable seeds the random number generator.

We want a value in the range `[0, 1]` because that's how the ratios are expressed in the `ConfigMap`. That is accomplished by dividing the value we get from `$RANDOM` by the maximum value we can get — note: `bash` doesn't do floating point math.

`awk` has a `rand` function, but it has to be seeded; we could also do this:

```sh
awk < /usr/local/e2e-targets \
    --assign "r=$RANDOM" \
    'BEGIN { srand(r); r = rand() } …'
```